optaplanner-benchmark/src/main/java/org/optaplanner/benchmark/impl/report/BenchmarkReport.java

```diff
@@ -74,6 +50,14 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import javax.imageio.ImageIO;
```

Comment:
(Suggestion, not important)
About import ordering: I believe the default import ordering in our default code style is different. Our default code style file is here:
  https://github.com/droolsjbpm/droolsjbpm-build-bootstrap/blob/master/ide-configuration/intellij-configuration/code-style/intellij-code-style_droolsjbpm-java-conventions.xml

Nevertheless, this is fine to merge, as sooner or later we do an "optimize imports" and it goes back to what it was :)